src/components/layout/PanelChange/index.tsx

```diff
@@ -262,6 +267,49 @@ export const PanelChange: FC<IProps> = ({
 		onClickStop(callback);
 	}, [callback, onClickStop]);
 
+	const getIbAllouRatio = async (address: string, price: string) => {
+		const contract = await getContract(address, ibAllouToken, web3);
+		const ratio = await contract.methods.growingRatio().call();
+		const ratioFloat = await fromWei(ratio, 18);
+		const ratioPrice = parseFloat(ratioFloat) * parseFloat(price);
+		return ratioPrice.toString();
+	};
+
+	function isIbAlluo(coinA: Coin, coinB: Coin, totalFlow: string) {
+		if (
+			coinA.includes(Coin.IbAlluoBTC) ||
+			coinA.includes(Coin.IbAlluoUSD) ||
+			coinA.includes(Coin.IbAlluoETH) ||
+			coinB.includes(Coin.IbAlluoBTC) ||
+			coinB.includes(Coin.IbAlluoUSD) ||
+			coinB.includes(Coin.IbAlluoETH)
+		) {
+			let tokenAddress: string = '';
+			let growthRatioPrice: string = '';
+			if (coinB === Coin.IbAlluoBTC) {
+				tokenAddress = IbAlluoBTCAddress;
+			} else if (coinB === Coin.IbAlluoETH) {
+				tokenAddress = IbAlluoETHAddress;
+			} else if (coinB === Coin.IbAlluoUSD) {
+				tokenAddress = IbAlluoUSDAddress;
+			}
+
+			if (tokenAddress !== '') {
```

Comment:
if !tokenAddress works just as well as tokenAddress !== '' saves a few chars and is cleaner